bundles/org.eclipse.equinox.http.servlet/src/org/eclipse/equinox/http/servlet/HttpSessionTrackerUtil.java

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
this class does not appear to be used anywhere! Can we remove it?

Reply:
It is here https://github.com/liferay/liferay-portal/blob/master/modules/apps/static/portal-osgi-web/portal-osgi-web-wab-extender/src/main/java/com/liferay/portal/osgi/web/wab/extender/internal/adapter/HttpAdapter.java#L169